packages/apputils/src/thememanager.ts

```diff
@@ -245,13 +259,22 @@ export class ThemeManager {
     return Promise.all([old, themes[theme].load()])
       .then(() => {
         this._current = theme;
-        Private.fitAll(this._host);
-        splash.dispose();
         this._themeChanged.emit({
           name: 'theme',
           oldValue: current,
           newValue: theme
         });
+
+        // Need to force a redraw of the app here to avoid a Chrome rendering bug/race condition
+        const style = this._host.node.getAttribute('style');
```

Comment:
Can we instead use `this._host.hide()` and `this._host.show()`, the phosphor commands that essentially do this display styling?

Reply:
(though using the phosphor commands may trigger expensive rerenders...)

Do you have a link to the Chrome bug report, so we can keep an eye on it to know when we can remove this workaround?

Reply:
I'll test out `this._host.hide()` and see if it works.

No bug report, just the results of my observations from testing. Typically, at least some scrollbars (especially the ones for the panels hosted on the left and right tab bars) won't update with the rest of theme, but can be fixed with a further refresh of the browser, or even just collapsing/uncollapsing the containing widget.

As a caveat, I'm still using Chrome 68 at the moment (they dropped proper support for multiple monitors on mac in later versions, so I've been loath to move on). It would be nice if someone could test the PR without the render fix on the latest chrome/not on a mac.

Reply:
- `this._host.hide()`/`show()` work just as well for fixing the rendering issue. My very not rigorous testing showed that `hide`/`show` took about 10% more rendering time per theme change than did directly setting `display`, but honestly it was probably well within the noise. Should I switch over to `hide`/`show`?

- I just tested this out on the latest version of Chromium (74), and the issue persists. Here's an example:

![image](https://user-images.githubusercontent.com/2263641/53281327-6bcc9300-36f4-11e9-8ce7-b63f6b9a8a16.png)

This is after changing from the Light theme to the Dark. While the launcher scrollbar changed colors correctly, the filebrowser scrollbar remained white. 

I took a quick look at Chromium's bug board for something similar, but didn't see anything similar. Given the way that the thememanager is currently dynamically loading the CSS, I'm going to go out on a limb here and guess that Jupyterlab is kind of an edge case here. I'll submit a bug report.

Reply:
> Should I switch over to `hide`/`show`?

I think so. It's definitely the more conventional "phosphor way" to do things, and may fix other things. Really, the phosphor elements *are* getting hidden and shown in the splash screen, and theme changes may affect measurements, so it's probably good to have a phosphor hide/show cycle so things can update.

Reply:
> theme changes may affect measurements, so it's probably good to have a phosphor hide/show cycle so things can update.

I guess that's probably what the fitAll call does, though. Still, I think it's probably not bad to do a hide/show cycle as a more conventional way to deal with the issue.

Reply:
The code's been switched to use `hide`/`show`.